source/server-selection/tests/README.rst

```diff
@@ -65,3 +65,36 @@ the TopologyDescription. Each YAML file contains a key for these stages of serve
 Drivers implementing server selection MUST test that their implementation
 correctly returns the set of servers in ``in_latency_window``. Drivers SHOULD also test
 against ``suitable_servers`` if possible.
+
+Selection Within Latency Window Tests
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+Each YAML file for these tests has the following format:
+
+- ``in_window``: array of servers in the latency window that the selected server
+  is to be chosen from. Each element will have all of the following fields:
```

Comment:
I think it would help implementors to add a TopologyDescription field  to the test setup similar to the existing `Server Selection Logic Tests`.  This would help drivers that need to create Topology/ServerDescriptions in order to test selection logic.

Reply:
Done. I added it as just another field in the file, separate from the `in_window` one, seeing as connection pool information is likely stored separate from the actual `ServerDescription` type within drivers. Let me know if this is what you had in mind.

Reply:
Nice, I'll test this out later today!